src/Components/ProgressBar/ProgressBar.tsx

```diff
@@ -14,22 +14,17 @@ const ProgressBar = ({ step }: ProgressBarProps) => {
   const { theme, formData } = useContext(Context);
   const totalSteps = getStepDirectory(formData.immutableReferrer).length + STARTING_QUESTION_NUMBER;
   const { id } = useParams();
-  let stepValue: number = 0;
-  if (step !== undefined) {
-    stepValue = step;
-  } else if (id !== undefined) {
-    stepValue = Number(id);
-  }
+
+  let stepValue = step || id || 0;
```

Comment:
Using `??` is better for this use case, because `||` will return the second argument if the first argument is `undefined`, `null`, `0` or `''`, while `??` will only return the second argument if the first one in `undefined` or `null`. 
```suggestion
  let stepValue = step ?? id ?? 0;
```

Reply:
Thanks for the explanation! I've reverted this.